### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by foolhardy21**
## Home Page

This PR has the UI Components for the homepage. This is only static, I have not added any dynamicity/feature yet.

- Files to review:
   - [ ] src/App.js
   - [ ] src/pages/Home.jsx
   - [ ] src/components/Home/Categories.jsx
   - [ ] src/components/Reusable/Header.jsx

- Use this link for preview - https://deploy-preview-4--stupefied-hawking-9910e7.netlify.app/

v-1
Thanks.

### CODE REVIEW

File: src/components/Home/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import nbaM from 'assets/nba-m.jpg'
+import nbaS from 'assets/nba-s.jpg'
+import sneakM from 'assets/sneak-m.jpg'
+import sneakS from 'assets/sneak-s.jpg'
+import styles from './home.module.css'
+
+const Categories = () => {
+
+    return (
+        <section className={`grid grid-maxcols-3 ${styles.categoryGrid} mg-top-md mg-btm-md`}>
```
Comment (vrx29) [inline src/components/Home/Categories.jsx:12]: You can try classnames npm package that can help you write clean with multiple classes intead of using template literals

File: src/pages/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import { Header } from "components/Reusable"
+import { useTheme } from "contexts"
+import { getBgColor, getTextColor } from "utils"
+
+const Home = () => {
+    const { theme } = useTheme()
+
+    return (
+        <div
+            style={{
```
Comment (vrx29) [inline src/pages/Home.jsx:11]: If not necessary try not to use inline styles

**Review by vrx29 — APPROVED** [review]
Everything looks good. You can merge it and continue

File: src/components/Home/Categories.jsx (hunk 1 above)
Comment (foolhardy21) [reply src/components/Home/Categories.jsx:12]: Thanks. I will surely look into it.

File: src/pages/Home.jsx (hunk 2 above)
Comment (foolhardy21) [reply src/pages/Home.jsx:11]: Sure, will change it.

**Review by Vishnu-Aithal — APPROVED** [review]
Good Code, Small PR , Nothing Much to Review , Good to Merge.
suggestions: Avoid Inline Styles

**Pull request merged**